README.md

```diff
@@ -23,6 +23,7 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (optional, required only for LLVM build; add to PATH)
 * [Python](https://www.python.org/downloads/) (optional, required only for LLVM build; add to PATH)
+* [Git] (https://git-scm.com/downloads) (Select "use Git from the Windows Command Prompt")
```

Comment:
@Zangetsu38 No. You can download the source code through Github and also go to submodules' pages and download their sources. Git is not required. And this is especially wrong, because this is the Windows port of the Git VCS. Please remove this, as this is not relevant for compiling, but is definitely a must have for development.